easybuild/easyconfigs/g/grib_api/grib_api-1.24.0-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,26 @@
+# File modified by MaKra - August 2017 (CSCS); adapted by @omlins (09.2018)
+easyblock = 'ConfigureMake'
+
+name = 'grib_api'
+version = '1.24.0'
+
+homepage = 'https://software.ecmwf.int/wiki/display/GRIB/Home'
+description = """The ECMWF GRIB API is an application program interface accessible from C, FORTRAN and Python
+ programs developed for encoding and decoding WMO FM-92 GRIB edition 1 and edition 2 messages. A useful set of
+ command line tools is also provided to give quick access to GRIB messages."""
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+
+source_urls = ['https://software.ecmwf.int/wiki/download/attachments/3473437/']
+sources = ['grib_api-%(version)s-Source.tar.gz']
+checksums = ['6b0d443cb0802c5de652e5816c5b88734cb3ead454eb932c5ec12ef8d4f77bcd']
+
+dependencies = [
+    ('JasPer', '1.900.1'), #2.x doesn't work (at least 2.0.14 doesn't)
```

Comment:
what do you mean by don't work ?

Reply:
It fails to build. I have seen the comment "#2.x doesn't work" in another recipe; so as it failed to build also in my case (with version 2.0.14), I added this comment as well and specified the version I have tried (version 2.0.14).